frontend/pkg/frontend/const.go

```diff
@@ -17,4 +17,7 @@ const (
 	PathSegmentDeploymentName    = "deploymentname"
 	PathSegmentActionName        = "actionname"
 	PathSegmentNodepoolName      = "nodepoolname"
+
+	HCPClusterResourceType = "hcpopenshiftclusters"
+	NodePoolResourceType   = "hcpopenshiftclusters/nodepools"
```

Comment:
Can you move the node pool constant to `internal/api/registry.go`?
We already have a constant for cluster resource type defined there.

Reply:
Done